### PR TITLE
[fix]: [CDS-38134]: Increase code coverage

### PR DESCRIPTION
**Pull request by nassergonzalez**
## Describe your changes
This change will increase the code coverage in areas where the code coverage was under 80%
## Checklist
- [x] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.
- [x] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/33644)

### CODE REVIEW

File: 960-api-services/src/test/java/io/harness/aws/AWSCloudformationClientTest.java [hunk 1]
```diff
@@ -480,26 +562,45 @@ public void checkServiceException() {
...
+
     service.describeStacks(
         region, describeStacksRequest, AwsInternalConfig.builder().accessKey(accessKey).secretKey(secretKey).build());
+
     service.getAllStackResources(region, describeStackResourcesRequest,
         AwsInternalConfig.builder().accessKey(accessKey).secretKey(secretKey).build());
+
     service.waitForStackDeletionCompleted(describeStacksRequest,
         AwsInternalConfig.builder().accessKey(accessKey).secretKey(secretKey).build(), region, null, 1000L);
-    verify(awsApiHelperService, times(8)).handleAmazonServiceException(any());
```
Comment (vlica-harness) [inline 960-api-services/src/test/java/io/harness/aws/AWSCloudformationClientTest.java:502]: I believe you intended to update the times in this line , but actually removed it.

File: bazelrc.common [hunk 2]
```diff
@@ -10,10 +10,10 @@ build:macos --javabase=@openjdk8u242_macos_archive//:runtime
 build:linux --host_javabase=@openjdk8u242_linux_archive//:runtime
 build:macos --host_javabase=@openjdk8u242_macos_archive//:runtime
 
-build --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_vanilla
```
Comment (vlica-harness) [inline bazelrc.common:13]: hey Nasser, seems the changes in this file should be reverted.

Comment (nassergonzalez) [reply bazelrc.common:13]: Yes, they are reverted now

File: 960-api-services/src/test/java/io/harness/aws/AWSCloudformationClientTest.java (hunk 1 above)
Comment (nassergonzalez) [reply 960-api-services/src/test/java/io/harness/aws/AWSCloudformationClientTest.java:502]: You are right, this should be fixed now

**Review by vlica-harness — APPROVED** [review]
> Reviewed 2 of 7 files at r1, 3 of 4 files at r2, 1 of 1 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33644)* status: :shipit: complete! ✓

**Review by tmacari-harness — CHANGES_REQUESTED** [review]
> Reviewed 2 of 7 files at r1, 2 of 4 files at r2, 1 of 1 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33644)* status: :shipit: complete! ✓

___
*[`125-cd-nextgen/src/test/java/io/harness/cdng/aws/service/AwsResourceServiceImplTest.java` line 186 at r4](https://reviewable.io/reviews/harness/harness-core/33644#-N3KJq5-5nMz4taARa6E:-N3KJq503eEo0wDnEX9c:b-ar2co9) ([raw file](https://github.com/harness/harness-core/blob/8d0309d591de488bd17619c396098a57b67b3414/125-cd-nextgen/src/test/java/io/harness/cdng/aws/service/AwsResourceServiceImplTest.java#L186)):*
> ```Java
>     Map<String, String> regions = service.getRegions();
>     assertThat(regions).size().isEqualTo(19);
>   }
> ```

Please add empty line.

**Review by tmacari-harness — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33644)* status: :shipit: complete! ✓

**Pull request merged**